ReliefApplications/ems-frontend
# Safe 499

# Description

Added the possibility for authorized admin users to see deleted records and either hard delete or restore them from Form and Resource view.

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

Created some dummy records and played around using all the functionalities in both views:
- Delete a record, switch view, restore it, switch view to confirm
- Delete a record, switch view, hard delete it, refresh to confirm

# Checklist:
